server/src/http/controllers/unsubscribeRecruteurLba.controller.ts

```diff
@@ -45,46 +44,57 @@ export default function (server: Server) {
         }
       }
 
-      const lbaCompaniesToUnsubscribe = await getDbCollection("recruteurslba").find(criteria).limit(ARBITRARY_COMPANY_LIMIT).toArray()
+      const lbaCompaniesToUnsubscribe = await getDbCollection("jobs_partners").find(criteria).limit(ARBITRARY_COMPANY_LIMIT).toArray()
 
       if (!lbaCompaniesToUnsubscribe.length) {
         result = { result: UNSUBSCRIBE_EMAIL_ERRORS.NON_RECONNU }
       } else if (lbaCompaniesToUnsubscribe.length > 1 && !sirets) {
         const companies = lbaCompaniesToUnsubscribe.map((company) => {
-          return { enseigne: company.enseigne, siret: company.siret, address: buildLbaCompanyAddress(company) }
+          return { enseigne: company.workplace_legal_name!, siret: company.workplace_siret!, address: company.workplace_address_label }
```

Comment:
```suggestion
          return { enseigne: company.workplace_legal_name, siret: company.workplace_siret, address: company.workplace_address_label }
```
Ca me paraît dangereux

Reply:
@remy-auricoste c'est systématiquement valorisé pour les recruteurs_lba d'après la raw_recruteurslba. je peux modifier le modèle attendu en sortie sinon de la route.

Reply:
oui, ce serait mieux de typer correctement plus en amont

Reply:
vu ensemble : modèle à usage interne dans jobsPartners dédié aux recruteurs LBA